resources/views/lesson/admin/index.blade.php

```diff
@@ -1,69 +1,109 @@
-@extends('layouts.app')
+@php use App\Models\Lesson;use App\Models\Registration; @endphp
+@extends('layouts.admin')
 
-@section('content')
+@section('head')
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+    <script src="{{ asset('js/admin/lesson/clickableRowsInTable.js') }}"></script>
+@endsection
+
+@section('admin_content')
     <div class="container">
         <div class="my-5 text-center">
-            <h1>{{__('customlabels.lesson_index_welcome')}}</h1>
+            <h1>{{__('lesson.admin_index_welcome')}}</h1>
         </div>
         <div class="my-5 row g2">
             <div class="col">
-                <h2>{{__('customlabels.lesson_index_statistics_tittle')}}</h2>
-                <b>{{__('customlabels.lesson_index_statistics_lesson_count')}}: </b> {{$lessons->count()}}
+                <h2>{{__('lesson.admin_index_statistics_tittle')}}</h2>
+                <b>{{__('lesson.admin_index_statistics_lessonCount')}}: </b> {{$lessons->count()}} <br>
+                @php($activeRegistrations = Registration::where('is_active', '=', true)->count())
+                <b>{{__('lesson.admin_index_statistics_totalActiveRegistrations')}}: </b> {{$activeRegistrations}}<br>
+                <b>{{__('lesson.admin_index_statistics_totalSpaces')}}: </b> {{$lessons->sum('total_signup_space')-$activeRegistrations}} <br>
+                <b>{{__('lesson.admin_index_statistics_totalFilledLessons')}}: </b> {{
+                    Lesson::whereHas('registrations', function ($query) {$query->where('is_active', true);})
+                    ->withCount(['registrations' => function ($query) {$query->where('is_active', true);}])
+                    ->get()
+                    ->filter(function ($lesson) {return $lesson->registrations_count == $lesson->total_signup_space;})
+                    ->count()
+                }}
+                <br>
+                <b>{{__('lesson.admin_index_statistics_totalAlmostFilledLessons')}}: </b> {{
+                    Lesson::whereHas('registrations', function ($query) {$query->where('is_active', true);})
+                    ->withCount(['registrations' => function ($query) {$query->where('is_active', true);}])
+                    ->get()
+                    ->filter(function ($lesson) {return (($lesson->registrations_count+2) >= $lesson->total_signup_space) && !($lesson->registrations_count == $lesson->total_signup_space);})
+                    ->count()
+                }}
+                <br>
             </div>
             @can('admin_panel')
```

Comment:
My changes to permissions may conflict with this btw

Reply:
Yes